lib/bundler/installer.rb

```diff
@@ -67,7 +68,7 @@ def run(options)
       install(options)
 
       lock unless Bundler.settings[:frozen]
-      generate_standalone(options[:standalone]) if options[:standalone]
+      Standalone.new(options[:standalone], @definition).generate if options[:standalone]
```

Comment:
I wonder if it would make more sense for this to be `Standalone.generate(options[:standalone], @definition)`. `Standalone` could then be turned into a `Module`.

Don't see much sense in keeping state in an object, when we throw the object away.